client/Main.hs

```diff
@@ -62,18 +65,53 @@ abortRunInfo = info getRunParser
  <> header "plutus-certification-client run abort — Abort a run"
   )
 
+getLogsParser :: Parser GetLogsArgs
+getLogsParser = GetLogsArgs
+  <$> getRunParser
+  <*> optional (option zonedTimeReader
+        ( long "after"
+       <> metavar "AFTER"
+       <> help "getting all the logs following a certain timestamp"
+        ))
+  <*> optional (option auto
+        ( long "action-type"
+       <> metavar "TYPE"
+       <> help "filter logs by action-type (Generate/Build/Certify)"
+        ))
+
+zonedTimeReader :: ReadM ZonedTime
+zonedTimeReader = do
+  urlStr <- str
+  case parseUrlPiece urlStr of
```

Comment:
I kept it in that format because one could easily just copy the `time-stamp` in as it was received from in the previous query of the logs and added it as an argument to receive just the new updates.
But I totally get your point